2013-03-24	Magnus Granberg		<zorry@gentoo.org>

		* gcc/gcc.c								include esp.h
		static const char *cc1_spec 			We set that in esp.h if ENABLE_ESP.

--- ./gcc/gcc.c	2010-01-21 10:29:30.000000000 -0500
+++ ./gcc/gcc.c	2010-01-29 23:29:16.000000000 -0500
@@ -44,6 +44,7 @@
 #include "opts.h"
 #include "params.h"
 #include "vec.h"
+#include "config/esp.h" /* for --enable-esp support */
 #include "filenames.h"
 
 /* By default there is no special suffix for target executables.  */
@@ -822,7 +823,9 @@
 
 static const char *asm_debug;
 static const char *cpp_spec = CPP_SPEC;
+#ifndef ENABLE_ESP
 static const char *cc1_spec = CC1_SPEC;
+#endif
 static const char *cc1plus_spec = CC1PLUS_SPEC;
 static const char *link_gcc_c_sequence_spec = LINK_GCC_C_SEQUENCE_SPEC;
 static const char *link_ssp_spec = LINK_SSP_SPEC;
